app/Models/Page.php

```diff
@@ -110,6 +110,26 @@ public function bake()
 		return $data;
 	}
 
+	/**
+	 * Get the published state of the page.
+	 * @return string One of 'new', 'draft', 'published'
+	 */
+	public function getStatusAttribute()
```

Comment:
Sam - fix this! Think about what you are doing man!

Reply:
@cfc7-unikent FIXED!